lib/fedora_migrate/repository_migrator.rb

```diff
@@ -52,6 +37,32 @@ def get_source_objects
 
     private
 
+    def migrate_object source
+      Logger.info "Migrating source object #{source.pid}"
+      result = begin
+        FedoraMigrate::ObjectMover.new(source, nil, options).migrate
+      rescue NameError => e
+        "The most likely explanation is that your target is not defined"
+        error_message(e)
+      rescue StandardError => e
+        error_message(e)
+      end
+      unless result == true
+        Logger.warn "#{source.pid} failed.\n#{result}"
+        @failed = @failed + 1
+      end
+    end
+
+    def migrate_relationship source
+      Logger.info "Migrating relationships for source object #{source.pid}"
+      begin
+        FedoraMigrate::RelsExtDatastreamMover.new(source).migrate
+      rescue StandardError => e
+        Logger.warn "#{source.pid} relationship migration failed.\n#{error_message(e)}"
```

Comment:
I was using warn because that's what I'm using elsewhere. It's also getting reported to STDERR that way. When I tried fatal, it was just getting written to the log.